robot_mechanism_controllers/src/joint_velocity_controller.cpp

```diff
@@ -136,7 +135,7 @@ void JointVelocityController::update()
   ros::Time time = robot_->getTime();
 
   double error = command_ - joint_state_->velocity_;
-  dt_ = time - last_time_;
+  dt_ = time - last_time_;  //will be 0.0 if starting() was called on this iteration
```

Comment:
This comment is actually wrong. `dt_` will be an epsilon value because of the time that passed between `starting` and `update`. This leads to a HUGE `i_error` that can never be compensated later on.